.travis.yml

```diff
@@ -1,76 +1,88 @@
 dist: trusty
-
+if: branch = master
 jobs:
   include:
-    - stage: build 1
+    - stage: build 1 - pyiron-base-image

```

Comment:
We should be consistent with the name of the image and the name of the stages. So here I would use `pyiron/base` rather than `pyiron-base-image`

Reply:
fixed the names via the last commit.